lib/puppet/provider/odl_user/idm.rb

```diff
@@ -0,0 +1,40 @@
+Puppet::Type.type(:odl_user).provide(:idm) do
+
+  commands :java => 'java'
+
+  def odl_path
+    '/opt/opendaylight'
+  end
+
+  def idm_cmd(*args)
+    java('-jar', "#{odl_path}/bin/aaa-cli-jar-0.5.0-SNAPSHOT.jar", '--dbd', odl_path, *args)
```

Comment:
Note I need to see if I can get Michael to rename this jar to eliminate the version name.